scripts/bench/benchmark.js

```diff
@@ -14,27 +14,29 @@ function wait(val) {
   return new Promise(resolve => setTimeout(resolve, val));
 }
 
-async function runScenario(benchmark, launcher) {
+async function runScenario(benchmark, chrome) {
+  const port = chrome.port;
   const results = await Lighthouse(
     `http://localhost:8080/${benchmark}/`,
     {
       output: 'json',
-      disableCpuThrottling: false,
```

Comment:
The config enables throttling by default
- https://github.com/GoogleChrome/lighthouse/blob/master/lighthouse-core/config/default-config.js#L50
- https://github.com/GoogleChrome/lighthouse/blob/master/lighthouse-core/config/perf-config.js#L12